sw/device/lib/dif/dif_entropy.c

```diff
@@ -93,12 +93,40 @@ dif_entropy_result_t dif_entropy_configure(const dif_entropy_t *entropy,
   return kDifEntropyOk;
 }
 
+inline static bool get_es_valid(const dif_entropy_t *entropy) {
+  uint32_t reg = mmio_region_read32(entropy->params.base_addr,
+                                    ENTROPY_SRC_INTR_STATE_REG_OFFSET);
+  return bitfield_bit32_read(reg, ENTROPY_SRC_INTR_STATE_ES_ENTROPY_VALID_BIT);
+}
+
+inline static void clr_es_valid(const dif_entropy_t *entropy) {
+  mmio_region_write32(entropy->params.base_addr,
```

Comment:
you can use `mmio_region_nonatomic_clear_bit32` here.

Reply:
yes..i just realized there's a world of wonderful functions i did not notice. Thanks for pointing them out.